src/Streamly/Internal/Data/Stream/IsStream/Transform.hs

```diff
@@ -865,6 +866,11 @@ nubBy :: -- (IsStream t, Monad m) =>
     (a -> a -> Bool) -> t m a -> t m a
 nubBy = undefined -- fromStreamD . D.nubBy . toStreamD
 
+{-# INLINE nubByMerge #-}
+nubByMerge :: (IsStream t, Monad m) =>
+    (a -> a -> Bool) -> t m a -> t m a
+nubByMerge eq = fromStreamD . D.nubByMerge eq . toStreamD
```

Comment:
Isn't this same as `uniqBy` above?

Reply:
Won't both of them be equivalent when the stream is sorted? Looks like, only the implementation is different, the algo is the same and the functionality is exactly the same.